src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -27,9 +28,13 @@ public partial class JsonWebToken : SecurityToken
         private string _encodedHeader;
         private string _encodedPayload;
         private string _encodedSignature;
+        private string _encodedToken;
         private string _encryptedKey;
         private string _initializationVector;
         private List<string> _audiences;
+#pragma warning disable CS0649 // Field 'JsonWebToken._encodedTokenMemory' is never assigned to, and will always have its default value
+        private ReadOnlyMemory<char> _encodedTokenMemory;
```

Comment:
Can we set this in the ctor and remove the #pragma?
We can also use the readonly keyword, this will stop accidental pointing to a new string.

Reply:
Now that a single code path applies to all .NET versions supported by Wilson the warning is no longer applicable